packages/enhanced_gradients/pubspec.yaml

```diff
@@ -13,7 +13,7 @@ dependencies:
   collection: ^1.17.1
   flutter:
     sdk: flutter
-  material_color_utilities: ^0.2.0
+  material_color_utilities: any
```

Comment:
Would our package really work with any version? We don't have test matrix for that. Let's add such

Reply:
Added (I think it works, we'll see)